src/openhim.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+const {registerMediator, activateHeartbeat} = require('openhim-mediator-utils')
+
+const logger = require('./logger')
+const config = require('./config')
+
+const mediatorConfigFile = fs.readFileSync(
+  path.resolve(__dirname, '..', 'mediatorConfig.json')
+)
+const mediatorConfigJson = JSON.parse(mediatorConfigFile)
+
+const configOptions = config.getConfig()
+
```

Comment:
Maybe a bit trivial, but we could prob just have these variables defined after each other and not have a blank line between them